app/controllers/clients_controller.rb

```diff
@@ -58,6 +56,6 @@ def client_params
       :state, :comments, :involved_authority, :competent_authority, :actual_activities,
       language_skills_attributes, relatives_attributes, schedules_attributes,
       contact_attributes
-      )
+    )
```

Comment:
I know, its unrelated. But rubocop hassles us every time and again. And it should have never been committed unlinted anyway...